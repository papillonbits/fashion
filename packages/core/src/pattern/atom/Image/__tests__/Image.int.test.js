import { large, medium, small, tiny } from './Image.int.story'

describe('<Image />', () => {
  describe('Render', () => {
    test('must match large', () => {
      expect(global.renderToJSON(large())).toMatchSnapshot()
    })

    test('must match medium', () => {
      expect(global.renderToJSON(medium())).toMatchSnapshot()
    })

    test('must match small', () => {
      expect(global.renderToJSON(small())).toMatchSnapshot()
    })

    test('must match tiny', () => {
      expect(global.renderToJSON(tiny())).toMatchSnapshot()
    })
  })
})
