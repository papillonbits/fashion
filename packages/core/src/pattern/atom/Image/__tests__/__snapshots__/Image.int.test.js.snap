// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> Render must match large 1`] = `
<div>
  <img
    alt="news"
    height="200"
    src="https://avatars.githubusercontent.com/u/81188645?s=200&v=4"
    width="200"
  />
</div>
`;

exports[`<Image /> Render must match medium 1`] = `
<div>
  <img
    alt="news"
    height="100"
    src="https://avatars.githubusercontent.com/u/81188645?s=200&v=4"
    width="100"
  />
</div>
`;

exports[`<Image /> Render must match small 1`] = `
<div>
  <img
    alt="news"
    height="50"
    src="https://avatars.githubusercontent.com/u/81188645?s=200&v=4"
    width="50"
  />
</div>
`;

exports[`<Image /> Render must match tiny 1`] = `
<div>
  <img
    alt="news"
    height="25"
    src="https://avatars.githubusercontent.com/u/81188645?s=200&v=4"
    width="25"
  />
</div>
`;
